client/state/selectors/test/get-post-revision-changes.js

```diff
@@ -1,134 +0,0 @@
-/** @format */
```

Comment:
@marekhrabe Is there really nothing to test here any longer or did I miss something in the rebase?